docs/src/main/sphinx/sql/explain-analyze.rst

```diff
@@ -42,38 +42,50 @@ relevant plan nodes). Such statistics are useful when one wants to detect data a
-                    clerk := tpch:clerk
+        Project[]
+        │   Layout: [clerk:varchar(15), count:bigint]
+        │   Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: 0B}
+        │   CPU: 8.00ms (3.51%), Scheduled: 63.00ms (15.11%), Blocked: 0.00ns (0.00%), Output: 1000 rows (28.32kB)
+        │   Input avg.: 15.63 rows, Input std.dev.: 24.36%
+        └─ Aggregate[type = FINAL, keys = [clerk], hash = [$hashvalue]]
+           │   Layout: [clerk:varchar(15), $hashvalue:bigint, count:bigint]
+           │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: 0B}
+           │   CPU: 8.00ms (3.51%), Scheduled: 22.00ms (5.28%), Blocked: 0.00ns (0.00%), Output: 1000 rows (37.11kB)
+           │   Input avg.: 15.63 rows, Input std.dev.: 24.36%
+           │   Collisions avg.: 0.00 (0.00% est.), Collisions std.dev.: ?%
+           │   count := count("count_0")
+           └─ LocalExchange[partitioning = HASH, hashColumn = [$hashvalue], arguments = ["clerk"]]
+              │   Layout: [clerk:varchar(15), count_0:bigint, $hashvalue:bigint]
+              │   Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: 0B}
+              │   CPU: 2.00ms (0.88%), Scheduled: 4.00ms (0.96%), Blocked: 23.15s (50.10%), Output: 1000 rows (37.11kB)
+              │   Input avg.: 15.63 rows, Input std.dev.: 793.73%
+              └─ RemoteSource[sourceFragmentIds = [2]]
+                     Layout: [clerk:varchar(15), count_0:bigint, $hashvalue_1:bigint]
+                     CPU: 0.00ns (0.00%), Scheduled: 0.00ns (0.00%), Blocked: 23.06s (49.90%), Output: 1000 rows (37.11kB)
+                     Input avg.: 15.63 rows, Input std.dev.: 793.73%
+
+    Fragment 2 [SOURCE]
+        CPU: 210.60ms, Scheduled: 327.92ms, Blocked 0.00ns (Input: 0.00ns, Output: 0.00ns), Input: 1500000 rows (18.17MB); per task: avg.: 1500000.00 std.dev.: 0.00, Output: 1000 rows (37.11kB)
+        Output layout: [clerk, count_0, $hashvalue_2]
+        Output partitioning: HASH [clerk][$hashvalue_2]
+        Aggregate[type = PARTIAL, keys = [clerk], hash = [$hashvalue_2]]
+        │   Layout: [clerk:varchar(15), $hashvalue_2:bigint, count_0:bigint]
+        │   CPU: 30.00ms (13.16%), Scheduled: 30.00ms (7.19%), Blocked: 0.00ns (0.00%), Output: 1000 rows (37.11kB)
+        │   Input avg.: 818058.00 rows, Input std.dev.: 0.00%
+        │   Collisions avg.: 8247.00 (25415.23% est.), Collisions std.dev.: 0.00%
+        │   count_0 := count(*)
+        └─ ScanFilterProject[table = hive:sf1:orders, filterPredicate = ("orderdate" > DATE '1995-01-01')]
+               Layout: [clerk:varchar(15), $hashvalue_2:bigint]
+               Estimates: {rows: 1500000 (41.48MB), cpu: 35.76M, memory: 0B, network: 0B}/{rows: 816424 (22.58MB), cpu: 35.76M, memory: 0B, network: 0B}/{rows: 816424 (22.58MB), cpu: 22.58M, memory: 0B, network: 0B}
+               CPU: 180.00ms (78.95%), Scheduled: 298.00ms (71.46%), Blocked: 0.00ns (0.00%), Output: 818058 rows (12.98MB), Physical Input: 4.51MB
```

Comment:
Seeing the example in docs, I can now see that this should actually be printed on the last line which prints `Input: ...`
This needs changes to `PlanPrinter#visitScanFilterAndProjectInfo` and `PlanPrinter#visitTableScan` instead of TextRenderer.